### PR TITLE
Added Spicy Vodka Pasta

### PR DESCRIPTION
**Pull request by yyxy-dev**
Added recipe to src/

### CODE REVIEW

**Comment by Knarkzel** [comment]
Looks good to me. Maybe missing the following?

```
- 🍽️ Servings: 4
```

File: src/spicy-vodka-pasta.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Spicy Vodka Pasta
+
+A tasty and flavorful pasta.
+
+- ⏲️ Prep time: 10 min
+- 🍳Cook time: 20 min
+
```
Comment (zyansheep) [inline src/spicy-vodka-pasta.md:7]:
```suggestion
- 🍽️ Servings: 4

```

File: src/spicy-vodka-pasta.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Ingredients (Makes 2)
+
+- 1/4 cup olive oil
+- 1 clove of garlic, diced
+- 1/4 of a green onion
+- 1/4 cup tomato paste
+- 1/4 cup heavy cream
+- 1 tablespoon vodka (alternatively, to taste)
+- 1 1/2 teaspoon red pepper flakes
+- 225g preferred pasta
```
Comment (zyansheep) [inline src/spicy-vodka-pasta.md:17]:
```suggestion
- 8 oz (225g) preferred pasta
```
You are using cups for everything else, it is kind of weird to suddenly shift to S.I. units

**Review by zyansheep — CHANGES_REQUESTED** [review]

File: src/spicy-vodka-pasta.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- 1 tablespoon butter
+- 1/4 cup parmesan cheese
+
+
```
Comment (hallzy) [inline src/spicy-vodka-pasta.md:21]:
```suggestion
- 1/4 cup parmesan cheese

```

File: src/spicy-vodka-pasta.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+2. Add tomato paste until darker in color / caramelized.
+3. Add heavy cream, vodka, and mix.
+4. Add pepper flakes (more = spicier).
+5. Stir until combined and remove from heat.
+6. Cook pasta in a separate pot until al dente, save 1/4 cup pasta water before draining.
+7. Add butter to sauce (while still hot/warm).
+8. Add pasta and pasta water to sauce.
+9. Add parmesan.
+
+
```
Comment (hallzy) [inline src/spicy-vodka-pasta.md:34]:
```suggestion
9. Add parmesan.

```

File: src/spicy-vodka-pasta.md (hunk 1 above)
Comment (yyxy-dev) [reply src/spicy-vodka-pasta.md:7]: It's actually 2 servings, let me fix it.

**Pull request closed without merging**